.github/workflows/wpt.yml

```diff
@@ -7,6 +7,20 @@ on:
   pull_request:
   push:
     branches: 'main'
+  workflow_dispatch:
+    inputs:
+      # TODO(#361): Use.
```

Comment:
what does it mean? could you include a full link to the issue and expand the comment?

Reply:
It's just a placeholder because these things seem to only work when they are checked into main.

These will become checkboxes in the workflow run dialog.

The intent is to use these in the next PR.

Reply:
In other words: it's not worth to elaborate here because this will either becoming self-documenting in the next PR, or be removed if it ends up not working.